src/plugins/pull-request.ts

```diff
@@ -61,6 +65,18 @@ export const createPullRequest = async (
       return true;
     }
 
+    const { stdout: currentBranch } = await asyncExec(
+      "git log --pretty=oneline -1"
+    );
+    const { stdout: mainBranch } = await asyncExec(
+      "git log --pretty=oneline -1 main"
+    );
+
+    if (currentBranch === mainBranch) {
```

Comment:
I do not mean to question your methods, because I have not tested this, but...

Depending on what branch you are on, the output could be different for the same commit hash:

```bash
b1929258c06bf419f4848ef66ca9c397729146c7 (HEAD -> nhcarrigan, upstream/main, main) chore: remove remnants of comment counting code (#49704)
b1929258c06bf419f4848ef66ca9c397729146c7 (HEAD -> main, upstream/main) chore: remove remnants of comment counting code (#49704)
```

Unless that is just my config spitting that out??

Reply:
This is a good catch - thanks!